doc/examples/segmentation/plot_metrics.py

```diff
@@ -0,0 +1,47 @@
+"""
+====================================================
+Comparison of segments
+====================================================
+
+In this example we will:
+ * compute adapted rand error as defined by the SNEMI3D contest
+ * find the variation of information between two segments
+ * find the split variation of information between two segments
+
+"""
+
+import matplotlib.pyplot as plt
+import numpy as np
+
+from skimage.data import coffee
+from skimage.measure import (compare_adapted_rand_error,
+                             compare_split_variation_of_information,
+                             compare_variation_of_information)
+from skimage.color import rgb2gray
+from skimage.filters import sobel
+from skimage.segmentation import felzenszwalb, slic, watershed
+from skimage.segmentation import mark_boundaries
+from skimage.util import img_as_float
+
+img = coffee()
+
+segments_slic = slic(img, n_segments=100, compactness=10, sigma=1)
+gradient = sobel(rgb2gray(img))
+segments_watershed = watershed(gradient, markers=2, compactness=0.001)
```

Comment:
There's a couple of `coins` examples that get basically a perfect segmentation. I suggest the following approach for this example:

- find the example where every coin is perfectly segmented. Run the same code here and call the result `im_true`. 
- find the compact watershed example where all the coins are vastly oversegmented. Call that `im_test1`.
- find another example where most of the coins are properly segmented but a few are background, or merged. Call that `im_test2`.
- find a final example where most of the coins are merged with the background. (Or maybe even an example where all the coins are merged with the background, ie `np.ones(coins.shape, dtype=int)`. But better if one or two coins are properly segmented. Call that `im_test3`.

Then, plot the following things:
- a scatter plot, with a legend, or text labels, plotting the `split_vi[0]` against `split_vi[1]` (or vice versa. The splits should be on the vertical axis, so that the compact watershed example is on the top-left corner, and the `np.ones` example is on the bottom right corner. See the section on VI in my PLoS ONE 2013 paper for what this plot looks like.)
- another scatter plot, with a legend, or text labels, plotting the precision against the recall for the three points.
- all four segmentations. (true, and the three tests.)

This all would fit quite nicely in a 2 row x 3 column figure.

Hopefully that's clear enough!